### PR TITLE
Proposing OpenFeature for Incubation

### PR DESCRIPTION
**Pull request by AloisReitbauer**
This PR proposes OpenFeature for incubation. 

There has been great momentum on the community and maintainer side. Since being accepted into sandbox earlier this year, we have reached 1.0 release status with support for major server-side languages and added an implementation for a cloud-native feature provider. Also, the lest of production adopters is growing daily. 

Currently, we are looking for a TOC sponsor

Alois and the OpenFeature maintainers team.

### CODE REVIEW

File: proposals/incubation/openfeature.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Propose OpenFeature to Incubation Stage
+
+
+
+
+## Introduction
+
++ Name: OpenFeature
++ Website: [openfeature.dev](https://openfeature.dev)
++ Source Control: [OpenFeature on Gitbh](https://github.com/open-feature)
```
Comment (lukaszgryglicki) [inline proposals/incubation/openfeature.md:10]: `OpenFeature on Gitbh` -> `OpenFeature on GitHub`.

File: proposals/incubation/openfeature.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
++ Code of Conduct: [OPenFeature Code of conduct](https://github.com/open-feature/community#code-of-conduct)
```
Comment (lukaszgryglicki) [inline proposals/incubation/openfeature.md:11]: `OPenFeature Code of conduct` -> `OpenFeature Code of conduct`.

File: proposals/incubation/openfeature.md [hunk 3]
```diff
@@ -54,10 +54,10 @@ Since achieveing sandbox we have progressed in the development of our SDKs:
...
 
 ##### Providers
-- .NET: Cloudbees, fladg, Go feature, LaunchDarkly, Split
-- GO: Cloudbees, fladg, Flipt, Go feature, Split
-- Java: Cloudbees, fladg, Flagsmith, Go feature, Split
-- JavaScript: Cloudbees, fladg, Go feature, LaunchDarkly, PostHog, Split
+- .NET: Cloudbees, fladg, Go feature flag, LaunchDarkly, Split
+- GO: Cloudbees, fladg, Flipt, Go feature flag, Split
+- Java: Cloudbees, fladg, Flagsmith, Go feature flag, Split
+- JavaScript: Cloudbees, fladg, Go feature flag, LaunchDarkly, PostHog, Split
```
Comment (craigbox) [inline proposals/incubation/openfeature.md:60]: s/fladg/flagd/ ?

File: proposals/incubation/openfeature.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+- Native support for metrics using Prometheus
+
+#### Provider Growth
+Providers are responsible for performing flag evaluations and provide an abstraction between the underlying flag management system and the OpenFeature SDK. Providers can wrap a vendor SDK, call a bespoke flag evaluation REST API, or even parse some locally stored file to resolve flag values. This allows the underlying flag evaluation logic to be changed without requiring a major code refactor.
+
+##### Providers
+- .NET: Cloudbees, fladg, Go feature flag, LaunchDarkly, Split
+- GO: Cloudbees, fladg, Flipt, Go feature flag, Split
+- Java: Cloudbees, fladg, Flagsmith, Go feature flag, Split
+- JavaScript: Cloudbees, fladg, Go feature flag, LaunchDarkly, PostHog, Split
```
Comment (lukas-reining) [inline proposals/incubation/openfeature.md:60]: ConfigCat and DevCycle are missing here

File: proposals/incubation/openfeature.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+#### SDK Progress
+Since achieveing sandbox we have progressed in the development of our SDKs:
+- [.NET](https://github.com/open-feature/dotnet-sdk) released v1.1.0
+- [Java](https://github.com/open-feature/java-sdk) released v1.2.0
+- [PHP](https://github.com/open-feature/php-sdk) released v2.0.0
+- [Python](https://github.com/open-feature/python-sdk) released v0.0.9
+- [JavaScript](https://github.com/open-feature/js-sdk) released v1.1.0
+- [GO](https://github.com/open-feature/go-sdk) released v1.2.0
+- [Ruby](https://github.com/open-feature/ruby-sdk) released v0.1.0
```
Comment (lukas-reining) [inline proposals/incubation/openfeature.md:40]: JS is already in 1.3.1 and 1.3.2 is staged

Comment (lukas-reining) [reply proposals/incubation/openfeature.md:40]: Most other SDKs also have newer versions, and for most of them the spec 0.6.0 update is coming.

File: proposals/incubation/openfeature.md (hunk 3 above)
Comment (beeme1mr) [reply proposals/incubation/openfeature.md]: The name of the tool is Go Feature Flag.

https://gofeatureflag.org/

**Comment by amye** [comment]
@cncf/cncf-toc - FYI

**Comment by DavidPHirsch** [comment]
@thomaspoignant could you please fix your DCO

**Pull request merged**